### PR TITLE
LookupServer: Load custom hostnames from `/etc/hosts`

### PR DESCRIPTION
**Pull request by christopherdumas**
Solves #186 .

### CODE REVIEW

**Comment by christopherdumas** [comment]
The DNS cache wasn't being used anywhere in the code anyway, so I just replaced it. Another PR can add it in again, properly working.

File: .clang_complete [hunk 1]
```diff
@@ -0,0 +1 @@
+-I~/serenity
```
Comment (awesomekling) [inline .clang_complete]: Looks like an accidental add.

File: Base/etc/LookupServer.ini [hunk 2]
```diff
@@ -1,3 +1,2 @@
 [DNS]
 IPAddress=8.8.8.8
-
```
Comment (awesomekling) [inline Base/etc/LookupServer.ini]: Seems unnecessary to touch this file in the commit just to remove a newline.

**Comment by christopherdumas** [comment]
Gotcha. I made sure it can merge with master, and also reintegrated it with the new reverse DNS lookup thing.

**Comment by awesomekling** [comment]
Looks all right. It's a bit funny that we cache x.x.x.x.in-addr.arpa as literal names alongside the other name, but it'll work for now :)

**Comment by awesomekling** [comment]
Btw check this out :)
http://egorsmirnov.me/2015/05/04/global-gitignore-file.html

**Pull request merged**